codes/ruby/chapter_heap/top_k.rb

```diff
@@ -0,0 +1,64 @@
+=begin
+File: top_k.rb
+Created Time: 2024-04-19
+Author: Blue Bean (lonnnnnnner@gmail.com)
+=end
+
+require_relative "./my_heap"
+
+### 元素入堆 ###
+def push_min_heap(heap, val)
+  # 元素取反
+  heap.push(-val)
+end
+
+### 元素出堆 ###
+def pop_min_heap(heap)
+  # 元素取反
+  -heap.pop
+end
+
+### 访问堆顶元素 ###
+def peek_min_heap(heap)
+  # 元素取反
+  -heap.peek
+end
+
+### 取出堆中元素 ###
+def get_min_heap(heap)
+  # 将堆中所有元素取反
+  heap.to_a.map { |x| x * -1 }
```

Comment:
```suggestion
  heap.max_heap.map { |x| -x }
```

Reply:
@bluebean-cloud Please delete the `MaxHeap#to_a` method and change to the attr_reader above. You have to make sure the consistency of codes first when adding an unusual method.

Reply:
I'm sorry for that, I have repaired it.
9a8f397973db6c1d159e182bdc4a7f48c2834ca7